h5p-bildetema/src/components/SearchPage/SearchResultCard/SearchResultCard.module.scss

```diff
@@ -9,6 +10,18 @@
   border: 2px solid $dark-beige;
 }
 
+.bookmarkButton {
+  position: absolute;
+  z-index: 10;
+  right: 1rem;
+
+  & button {
+    all: unset;
+    padding: 0.5rem;
+    cursor: pointer;
+  }
+}
+
```

Comment:
Add background color and hover state to the bookmark button in this pr?

Reply:
svgs sometimes get extra space under because they are inline, so setting `display: block` on the svg can get rid of that. It makes it easier to get the same padding around the button.